libs/contract-core/package.json

```diff
@@ -32,9 +31,12 @@
     "clean:contracts": "rimraf ./src/contracts",
     "clean:types": "rimraf ./src/types && rimraf ./migrations/types",
     "test:contract": "truffle test",
-    "prod:test": "cross-env CONFIG_NAME=prodtest yarn _deploy",
-    "dev:live": "yarn wait-port 9545 && cross-env CONFIG_NAME=devlive yarn _deploy",
-    "_deploy": "yarn contract-migrate && yarn contract-verify",
+    "prod:test": "cross-env CONFIG_NAME=prodtest yarn deploy",
+    "dev:live": "yarn wait-port 9545 && cross-env CONFIG_NAME=devlive yarn deploy:polygon",
+    "deploy": "runs-s deploy:*",
+    "deploy:polygon": "truffle migrate --network polygon",
```

Comment:
I think I got a bit carried away with centralizing scripts here.  Was more efficient to leave deployement in this file